src/providers/FusionAuthProvider.tsx

```diff
@@ -1,95 +1,138 @@
-import React, { useCallback, useContext, useMemo } from 'react';
+import React, {
+    useCallback,
+    useContext,
+    useEffect,
+    useMemo,
+    useState,
+} from 'react';
 import { TextEncoder } from 'util';
+import axios from 'axios';
 
 export interface IFusionAuthContext {
     login: (state: string) => Promise<void>;
     logout: () => Promise<void>;
     register: (state: string) => Promise<void>;
+    user: Record<string, any>;
 }
 
 export const FusionAuthContext = React.createContext<IFusionAuthContext>({
     login: () => Promise.resolve(),
     logout: () => Promise.resolve(),
     register: () => Promise.resolve(),
+    user: {},
 });
 
-interface Props {
+export interface IFusionAuthConfiguration {
```

Comment:
Let's get rid of the `I` prefix on this as well. It is only used in the case of context providers so it doesn't conflict with the provider name. In this case there is nothing it conflicts with, so is simpler to just use `FusionAuthConfig`
```suggestion
export interface FusionAuthConfig {
```